app/views/conversations/_gpt_sidebar.html.erb

```diff
@@ -0,0 +1,13 @@
+  <div class="h-10 flex pr-2 rounded-lg hover:bg-zinc-400 hover:bg-opacity-25 justify-between items-center group cursor-pointer mb-1 mr-5">
+    <div id="profile" class="ml-2 mr-2 w-7 h-7 bg-white rounded-full"></div>
+    <div class="flex-1 text-white truncate">
+      <%= title %>
+    </div>
+    <div class="<%= defined?(selected) && selected ? 'flex' : 'hidden' %> group-hover:flex gap-2 pl-2">
```

Comment:
> `<%= defined?(selected) && selected ? 'flex' : 'hidden' %>`

This is one area where tailwind gets messy.

Reply:
Agreed, I would like a more elegant syntax for conditionally including a single value. I tried branching on the overall div, meaning:

```
if ….
   div
else
   div
```

But it’s too much duplication when I have 10 classes and I’m only varying a single one. It’s too easy for them to drift over time. Actually, maybe a better syntax is:

1. Using strict locals so that `selected` always exists and is initialized with a default value if it’s undefined.
2. Rewriting this line to a single override: `<div class=“flex <%=` !selected && `hidden’` %> …`

This works because `hidden` will override `flex`, but there are still going to be cases where I’ll need a tertiary operator. I don’t have a solution for that.

Reply:
Or, maybe I just write a custom tertiary helper to make it a bit simpler to inline these:

<%= if_true(selected, “flex”, “hidden”) %>

Reply:
I considered both of these suggestions but anything I could think of felt out of scope. I don't know if the value of a helper is really worth the effort yet.